go/v1beta1/storage/esutil/types.go

```diff
@@ -79,19 +80,34 @@ type EsIndexDocError struct {
 // Elasticsearch /_delete_by_query response
 
 type EsDeleteResponse struct {
-	Deleted int `json:"deleted"`
+	Took                 int           `json:"took"`
+	TimedOut             bool          `json:"timed_out"`
+	Total                int           `json:"total"`
+	Deleted              int           `json:"deleted"`
+	Batches              int           `json:"batches"`
+	VersionConflicts     int           `json:"version_conflicts"`
+	Noops                int           `json:"noops"`
+	ThrottledMillis      int           `json:"throttled_millis"`
+	RequestsPerSecond    float64       `json:"requests_per_second"`
+	ThrottledUntilMillis int           `json:"throttled_until_millis"`
+	Failures             []interface{} `json:"failures"`
 }
 
 // Elasticsearch /_bulk query fragments
 
 type EsBulkQueryFragment struct {
-	Index *EsBulkQueryIndexFragment `json:"index"`
+	Index  *EsBulkQueryIndexFragment  `json:"index"`
+	Create *EsBulkQueryCreateFragment `json:"create"`
 }
 
 type EsBulkQueryIndexFragment struct {
 	Index string `json:"_index"`
 }
 
+type EsBulkQueryCreateFragment struct {
```

Comment:
We might be able to merge this with the other fragment struct by using `omitempty` in the json tag on both fields. But I agree that it doesn't make much of a difference for now since there's no overlap.